pagopa/io-app
# chore: [IOCOM-1672] InApp Browser checking on FIMS

⚠️ This PR depends on [io-react-native-login-utils#16](https://github.com/pagopa/io-react-native-login-utils/pull/16) ⚠️

## Short description
This PR adds InApp Browser checking on the FIMS authorisation flow.

## List of changes proposed in this pull request
- `login-utils` library updated to last spec (1.0.5)
- `inAppBrowserSupported` check added on `handleFimsGetConsentsList` saga startup. 

## How to test
Using the io-dev-api-server, the FIMS flow:
- on iOS, the InApp Browser is always supported, so the authorisation flow should always start
- on Android, disable any browser listed in the App settings section. The authorisation flow should report the unsupported InApp Browser
